core/arch/arm/include/kernel/generic_boot.h

```diff
@@ -53,4 +54,6 @@ void *get_embedded_dt(void);
 /* Returns external DTB if present, otherwise NULL */
 void *get_external_dt(void);
 
+unsigned long get_aslr_seed(void *fdt);
```

Comment:
We need a stub for `!CFG_DT` otherwise there is a link error (HiKey960 for instance). Of course it doesn't make much sense to enable ASLR without DT since the only source for the seed is the DT, but it's still useful for testing and also we could have another (platform-specific) seed source.

Reply:
...actually we need it in `generic_boot.c`, not here, since it's called from assembler.

Reply:
Good point, I'll fix